cmd/test_util.go

```diff
@@ -10,17 +10,24 @@ import (
 
 var path = "../fixtures/"
 
-func runTest(t *testing.T, file string, function interface{}, errCode int, imageStr ...string) {
+func runTest(t *testing.T, file string, function interface{}, errCode int, argStr ...string) {
 	assert := assert.New(t)
 	file = filepath.Join(path, file)
 	var image imgFlags
+	var limits limitFlags
 	switch function.(type) {
 	case (func(imgFlags, Items) []Result):
-		if len(imageStr) != 1 {
+		if len(argStr) != 1 {
 			log.Fatal("Incorrect number of images specified")
 		}
-		image = imgFlags{img: imageStr[0]}
+		image = imgFlags{img: argStr[0]}
 		image.splitImageString()
+	case (func(limitFlags, Items) []Result):
+		if len(argStr) != 2 {
```

Comment:
The test doesn't always need limit parameters

Reply:
For the arguments, it would be nice to be able to name them. But this is not the subject of this PR.
To be simple, the code accepts 0 or 2 arguments.